CMakeLists.txt

```diff
@@ -249,6 +249,7 @@ endif()
 
 add_subdirectory (geom_mgr)
 add_subdirectory (regridder_mgr)
+add_subdirectory (hconfig)
```

Comment:
`hconfig_utils` might be better.   Not worth a lot of back and forth though.

Reply:
Done